dbcog/token_mappings.py

```diff
@@ -87,6 +87,8 @@ class EvoTypes(Enum):
     EvoTypes.NONPIXEL: ('nonpixel', 'np'),
 }
 
+# awakening view state & awakening help props in padinfo are transforming this dict to `int: tuple`
+# so rewrite those two if this gets refactored at some point in the future!
```

Comment:
This is kind of awful and I'd really like to not do this but I'm not sure how to avoid given the cross-cog import limitations